Create pythonjobsie.yml

mariusavram91 opened the pull request:
Added Python Jobs Ireland, a Django Jobs board for Ireland.

mariusavram91 committed:
Create pythonjobsie.yml

Added Python Jobs Ireland, a Django Jobs board for Ireland.

shiftkey commented on _data/projects/pythonjobsie.yml:
```diff
@@ -0,0 +1,12 @@
+name: pythonjobs.ie
+desc: Python Jobs Ireland: find or post Python related jobs/A Django Jobs board.
```
I'm pretty sure the `/` here is why CI is currently failing...

mariusavram91 committed:
Rephrase description

The description for pythonjobs.ie has been rephrased, "/" and ":" have been removed.

mariusavram91 commented:
Hi Brendan, thanks for that! I fixed it.

shiftkey commented:
@mariusavram91 thanks!
